src/commands/build/build_project.py

```diff
@@ -83,7 +83,7 @@ def build_project(
                     + '\nDid you forget to update protostar.toml::["protostar.contracts"]?'
                 )
             ) from err
-        except (StarkException, VmException, PreprocessorError) as err:
+        except (StarkException, VmException, PreprocessorError) as err: #type: ignore
```

Comment:
Is type ignoring necessary?

Reply:
I don't know why type checker raises error here, it seem correct, do you have any idea?
```
error: Except clause is unreachable because exception is already handled (reportGeneralTypeIssues)
```

Reply:
It's a bug in `pyright`. It's fixed in newer versions. Run `poetry install`.